configure.d/1_blk_queue_exit_for_rhel.conf

```diff
@@ -12,7 +12,7 @@ LINUX_MAJOR_SINCE=4
 LINUX_MINOR_SINCE=18
 LINUX_SUB_SINCE=0
 RHEL_MAJOR_SINCE=348
-RHEL_MINOR_SINCE=2
+RHEL_MINOR_SINCE=7
```

Comment:
Have you considered limiting the scope of workaround up to some RHEL version? I'm not sure what would be the right approach. If this gets fixed at some point in a newer version of RHEL kernel 4.18.0 then CAS will start failing due to incorrect application of the workaround. 

Maybe we should enable workaround only on this specific kernel version and fail compilation on any newer RHEL 4.18.0 kernel? We'll still be working with newer kernels via weak modules mechanism, so there will be some limited room for safe upgrade.

Reply:
As discussed, we should leave it as it is and file a bug to RH. We would then specify the lowest and highest version numbers when (or if at all) it's fixed.